src/init.cpp

```diff
@@ -521,8 +521,30 @@ void SetupServerArgs()
     hidden_args.emplace_back("-daemon");
 #endif
 
+    //
+    // Elements-specific arguments.
+    //
+
+    std::vector<std::string> elements_hidden_args = {"-con_fpowallowmindifficultyblocks", "-con_fpownoretargeting", "-con_nsubsidyhalvinginterval", "-con_bip16exception", "-con_bip34height", "-con_bip65height", "-con_bip66height", "-con_npowtargettimespan", "-con_npowtargetspacing", "-con_nrulechangeactivationthreshold", "-con_nminerconfirmationwindow", "-con_powlimit", "-con_bip34hash", "-con_nminimumchainwork", "-con_defaultassumevalid", "-npruneafterheight", "-fdefaultconsistencychecks", "-fmineblocksondemand", "-fallback_fee_enabled", "-pchmessagestart"};
+
+    gArgs.AddArg("-pubkeyprefix", strprintf("The byte prefix, in decimal, of the chain's base58 pubkey address. (default: %d)", 235), false, OptionsCategory::ELEMENTS);
+    gArgs.AddArg("-scriptprefix", strprintf("The byte prefix, in decimal, of the chain's base58 script address. (default: %d)", 75), false, OptionsCategory::ELEMENTS);
+    gArgs.AddArg("-secretprefix", strprintf("The byte prefix, in decimal, of the chain's base58 secret key encoding. (default: %d)", 239), false, OptionsCategory::ELEMENTS);
+    gArgs.AddArg("-extpubkeyprefix", strprintf("The 4-byte prefix, in hex, of the chain's base58 extended public key encoding. (default: %s)", "043587CF"), false, OptionsCategory::ELEMENTS);
+    gArgs.AddArg("-extprvkeyprefix", strprintf("The 4-byte prefix, in hex, of the chain's base58 extended private key encoding. (default: %s)", "04358394"), false, OptionsCategory::ELEMENTS);
+    gArgs.AddArg("-bech32_hrp", strprintf("The human-readable part of the chain's bech32 encoding. (default: %s)", "bc"), false, OptionsCategory::ELEMENTS);
```

Comment:
because we haven't picked one for elements yet!

In general I expect us to take the network name specifically in the `hrp`, like `liquidv`, but for elements it can be more generic.

`ele`?

Reply:
I think the base58 defaults are regtest's if I'm not mistaken. So perhaps this should be `"bcrt"`?

Reply:
Ah no this is not for parent. Yeah this is because we don't have one for elements yet.